### DISCUSSION TITLE
Add adapter support for other UI Libraries other than Material Ui

### DISCUSSION THREAD

**Discussion by torver213** [react-page/react-page#1203] [General] [answered]
Kudos to the creators and maintainers of this library. You people have done a wonder job.

Is there any way to use another custom UI library other than Material Ui? or would there be an adapter that can help us to achieve this? For instance in my current project, i make use of @mui although it is built on top of Material UI but some components differ and in trying to use @react/page in my project,  @mui and Material UI are conflicting making my UI look ugly. Any suggestion will be appreciated. Else maybe i will have to rewrite the library and change it from material Ui to @mui to support my current project.

**Comment by macrozone** [comment, marked as answer]
> Kudos to the creators and maintainers of this library. You people have done a wonder job.
> 
> Is there any way to use another custom UI library other than Material Ui? or would there be an adapter that can help us to achieve this? For instance in my current project, i make use of @mui although it is built on top of Material UI but some components differ and in trying to use @react/page in my project, @mui and Material UI are conflicting making my UI look ugly. Any suggestion will be appreciated. Else maybe i will have to rewrite the library and change it from material Ui to @mui to support my current project.

react-page version 5 now uses mui (version 5) and should not conflict with the rest of your app. You can also override the theme for react-page to match your ui. I suggest to update! 

completely make it independent of any ui library will be hard and will hinder innovation. Instead I advice to leaverage theming

**Comment by torver213** [comment]
Thank you for your swift response. This is exactly what I'm looking for. I will update my packages asap.

**Reply by macrozone** [reply]
i update the release notes with some details: https://github.com/react-page/react-page/releases/tag/v5.0.0

concerning function or api there is no breaking change

**Reply by macrozone** [reply]
> Kudos to the creators and maintainers of this library. You people have done a wonder job.

thank you for your kind words!

**Comment by iPagar** [comment]
I made a local react-page copy and updated components with [NextUI](https://nextui.org) and it looks beautiful! ✨
<img width="1512" alt="Screenshot 2022-07-10 в 13 28 13" src="https://user-images.githubusercontent.com/10209901/178139150-5f74a414-3292-4401-ba77-8d6c9485b734.png">
<img width="1512" alt="Screenshot 2022-07-10 в 13 27 41" src="https://user-images.githubusercontent.com/10209901/178139146-42df84b2-60b5-4ee2-bc6d-8ab38d5916db.png">
